utils/sb2-config-gcc-toolchain

```diff
@@ -226,7 +226,7 @@ if [ $? != 0 ]; then
 	exit 1
 fi
 
-SBOX_CROSS_GCC_DIR=$(dirname $(which $GCC))
+SBOX_CROSS_GCC_DIR=$(readlink -f $(dirname $(which $GCC)))
```

Comment:
Quote this please.

Reply:
done

Reply:
Please also quote the dirname and which commands.